### ISSUE TITLE
Бухгалтерия. Доработка Типовых хозяйственных операций

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#2913]
Originally reported on Google Code with ID 2913

```

  Исследователь > Бухгалтерия > Типовые хозяйственные операции.
  В диалоговое окно "Типовой операции" добавьте галку "Запись отключена".
  Данная галка будет удобна в использовании, когда пользователь из перечня "Типовых
операций" выбирает определённую "Типовую операцию". Например, когда у пользователя
доступна вся ветка неких ТХО, включая и родительские ТХО. Родительские ТХО, как обычно
не содержат "Типовых проводок". Как следствие, пользователь выбрав родительскую ТХО
не получит сформированных проводок по той или иной хозяйственной операции (документу).
В результате всего: ошибки в данных, на устранение которых уходит время.

```

Reported by `NikolayUkleyko` on 2012-11-05 15:10:32

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2012-11-29 10:14:58
- **Labels added**: Type-Enhancement, Priority-Medium, GedeminEXE, Accountancy

**Comment by gsbelarus** [comment]
Reported by `NikolayUkleyko` on 2013-06-11 07:30:50
- **Status changed**: `Duplicate`
- **Merged into**: #3149
